code/drasil-example/Drasil/DblPendulum/Assumptions.hs

```diff
@@ -30,7 +30,7 @@ cartCoordRightDesc = S "The" +:+ phrase cartesian `sIs` S "right-handed where po
                          phrase xAxis `sAnd` phrase yAxis +:+ S "point right up"
 
 yAxisDirDesc :: Sentence
-yAxisDirDesc = S "The direction" `ofThe'` phrase yAxis `sIs` S "directed opposite to" +:+. phrase gravity
+yAxisDirDesc = S "direction" `ofThe'` phrase yAxis `sIs` S "directed opposite to" +:+. phrase gravity
```

Comment:
This is very odd. The same kind of sentence in the `Projectile` example has the `The` in the sentence. So I wonder what is different here, i.e. where / why does it get added uniformly? Can you please track that down?

Reply:
I think it might be because of the `ofThe` function since it adds an extra "the" before joining its arguments with the words "of the".
```haskell
ofThe  a b = S "the" +:+ sentHelper "of the" a b
```
Most of the projectile sentences don't use `ofThe` at the beginning except for this one:
```haskell
yAxisGravityDesc :: Sentence
yAxisGravityDesc = S "direction" `ofThe'` phrase yAxis `sIs` S "directed opposite to" +:+. phrase gravity
```
which also does not include a literal "the" at the beginning.

Reply:
Oh, that's weird. And not the expected behaviour. I would changed `ofThe` to stop doing that, and create a differently named combinator for that.  So the correct fix would be 
1. to create that new combinator that does what `ofThe` currently does
2. change the current correct uses of `ofThe` to use that new one
3. have `ofThe` do what's expected
4. use `ofThe` appropriately
5. put back those initial "the" (or simply revert this PR and restart).

Note that we want to minimize explicit capitalization of words, and let the framework deal with it. Take a further look at Projectile as an example.

Reply:
I agree many of the use cases for `ofThe` currently make sense, it's just some few sentences that read weirdly because of the naming of the combinator. There's also similar functions in `Utils.Drasil.Sentence` such as `ofThe'` (capital variant),`ofGiv`, `ofGiv'`, and `isExpctToHv`. So would making those combinators into something slightly different such as `theOfThe` and `theOfGiv` work well? After that, then should I away the extra "the" from `ofThe` and manually change the ones that will make more sense?

With that being said, is there an easy way to rename a function in the whole repo? I've used VSCode to rename all functions in an individual file before, but I was wondering if there was a better way to go about it for a larger project like this. I think it might be easier to merge the PR and then I'll search and replace all `ofThe` badly acting functions on a new PR.

Also, for minimizing explicit capitalization, I think that `ofThe'` currently handles that better than putting a literal "The" at the beginning of the sentence. The projectile definition still makes enough sense without the explicit "The" at the beginning. Changing this might make it less readable if we don't include the explicit "The" when using the 'new' definition of `ofThe`.

Reply:
Perhaps `the-ofThe` might be even better, if `-` is allowed in a name. `the_ofThe` otherwise. I forget the exact lexical conventions of Haskell.  And then yes, get right of the extra "the" in `ofThe` and adjust accordingly.

Renaming all functions in a repo is a function of the editor / IDE you use. The one I use wouldn't help me... but maybe @balacij uses something cleverer?  There is a Haskell plugin for VSCode.

Reply:
Okay, good to know. Thanks!